component/Meetings/MeetingsList.jsx

```diff
@@ -17,19 +18,22 @@ import { openNotificationBox } from "../common/notification";
 import { DateBox } from "../DashBoard/component/helperComponent";
 import MeetingListSkeleton from "./component/MeetingListSkeleton";
 import { CASUAL_MEETINGTYPE, GOAL_TYPE, REVIEW_TYPE } from "./constants";
+import { MemberListHook } from "../../component/common/hooks";
 
 const currentTime = moment().format();
 
 function MeetingsList({ user }) {
   const [loading, setLoading] = useState(false);
   const [meetingsList, setMeetingsList] = useState([]);
+  const [filterId, setFilterId] = useState("ALL");
+  const { membersList } = MemberListHook(user);
 
   async function fetchMeetingList() {
     setLoading(true);
     setMeetingsList([]);
 
     await httpService
-      .get(`/api/meetings`)
+      .get(`/api/meetings?filterId=${filterId}`)
       .then(({ data: response }) => {
```

Comment:
better to ignore .then (). when inside a asyc function. retreive value directly

Reply:
okay will try this from now on